PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -165,20 +165,13 @@ LanguageNameInfo ScintillaEditView::_langNameInfoArray[L_EXTERNAL + 1] = {
 
 int getNbDigits(int aNum, int base)
 {
-	int nbChiffre = 1;
-	int diviseur = base;
+	int nbChiffre = 0;

```

Comment:
Why not take the opportunity to convert the code to English at this time?
Suggest `nbChiffre` become `nbDigits`.

Reply:
You are right. As this is my first contribution, I wanted to change as little as possible. I found more bugs, so I'll know better next time.

Reply:
Changed identifier as suggested
Fixed author

Reply:
@donho Hopefully it is ok for me to make the suggestion to convert French to English in the code.  I know that when I submitted some Column Editor code changes previously, I did such language updates and it was accepted...

Reply:
@alankilborn @cddiffz 
Yes, it's good. 
I did it (using inconsistently French word) because both are not my mother tongue.
However, the code should be done by a international language for the sake that everyone can understand so everyone can maintain it.